src/app/components/bot/PlatformSelect.module.css

```diff
@@ -24,9 +24,6 @@
   gap: 16px;
   padding: 16px;
 
-  /* position: absolute; */
-  z-index: 100;
```

Comment:
this was causing a linter error since it should be using a var, but removing completely because z-index only applies to positioned elements